### PR TITLE
Brush up some fist attack automation

### PR DESCRIPTION
**Pull request by simonward**
Closes https://github.com/foundryvtt/pf2e/issues/12986

### CODE REVIEW

File: packs/feats/arcane-fists.json [hunk 1]
```diff
@@ -26,35 +26,15 @@
...
-                        "dice": 1,
-                        "die": "d6"
-                    }
+                "fist": true,
+                "key": "Strike"
+            },
+            {
+                "key": "DamageDice",
+                "override": {
+                    "dieSize": "d6"
```
Comment (stwlam) [inline packs/feats/arcane-fists.json:35]:
```suggestion
                "override": {
                    "upgrade": true
                },
                "predicate": [
                    "item:category:unarmed",
                    "item:damage:die:faces:4"
                ]
```

File: packs/feats/arcane-fists.json [hunk 2]
```diff
@@ -26,35 +26,15 @@
...
-                ],
-                "range": null,
-                "slug": "fist",
-                "traits": [
-                    "agile",
-                    "finesse",
-                    "nonlethal",
-                    "unarmed"
-                ]
+                "selector": "fist-damage"
```
Comment (stwlam) [inline packs/feats/arcane-fists.json:37]:
```suggestion
                "selector": [
                    "fist-base-damage"
                ]
```

File: packs/feats/arcane-fists.json [hunk 3]
```diff
@@ -64,6 +44,18 @@
                 "mode": "add",
                 "property": "weapon-traits",
                 "value": "arcane"
+            },
+            {
+                "key": "CriticalSpecialization",
+                "predicate": [
+                    "item:group:brawling",
```
Comment (stwlam) [inline packs/feats/arcane-fists.json:51]:
```suggestion
                    "item:category:unarmed",
                    "item:group:brawling",
```

File: packs/feats/ferrousoul.json [hunk 4]
```diff
@@ -26,27 +26,33 @@
...
-                        "dice": 1,
-                        "die": "d6"
-                    }
+                "fist": true,
+                "key": "Strike"
+            },
+            {
+                "key": "DamageDice",
+                "override": {
+                    "dieSize": "d6"
```
Comment (stwlam) [inline packs/feats/ferrousoul.json:35]: Similar as the above

File: packs/feats/hand-of-the-lich.json [hunk 5]
```diff
@@ -30,27 +30,34 @@
...
-                        "die": "d6"
-                    }
+                "fist": true,
+                "key": "Strike"
+            },
+            {
+                "key": "DamageDice",
+                "override": {
+                    "damageType": "void",
+                    "dieSize": "d6"
```
Comment (stwlam) [inline packs/feats/hand-of-the-lich.json:40]: Same

File: packs/feats/mummy-dedication.json [hunk 6]
```diff
@@ -45,26 +45,24 @@
...
-                        "dice": 1,
-                        "die": "d6"
-                    }
+                "fist": true,
+                "key": "Strike"
+            },
+            {
+                "key": "DamageDice",
+                "override": {
+                    "dieSize": "d6"
```
Comment (stwlam) [inline packs/feats/mummy-dedication.json:54]: same

File: packs/heritages/warrior-automaton.json [hunk 7]
```diff
@@ -18,36 +18,15 @@
...
-                        "dice": 1,
-                        "die": "d6"
-                    }
+                "fist": true,
+                "key": "Strike"
+            },
+            {
+                "key": "DamageDice",
+                "override": {
+                    "dieSize": "d6"
```
Comment (stwlam) [inline packs/heritages/warrior-automaton.json:27]: same

File: packs/feats/arcane-fists.json (hunk 3 above)
Comment (simonward) [reply packs/feats/arcane-fists.json:57]: Are you certain? The description says "At 5th level, you also gain the critical specialization effects of unarmed attacks in the brawling group **and weapons** in the brawling group."

Comment (stwlam) [reply packs/feats/arcane-fists.json:57]: Nope, my mistake: ignore

**Pull request merged**